molgenis-security/src/main/java/org/molgenis/security/settings/AuthenticationSettingsImpl.java

```diff
@@ -80,21 +80,7 @@ public void init() {
           .setDescription(
               "Enable or enforce users to sign in with Google Authenticator. "
                   + "Must be disabled if you specify OpenID Connect authentication servers")
-          .setValidationExpression(getSignIn2FAValidationExpression());
-    }
-
-    /**
-     * 2FA should be disabled if there are any OIDC clients enabled.
-     *
-     * @return true if condition is met
-     */
-    private static String getSignIn2FAValidationExpression() {
-      return String.format(
-          "newValue(!($('%s').value() && $('%s').value().length)).or($('%s').eq('%s')).value()",
-          OIDC_CLIENTS,
-          OIDC_CLIENTS,
-          SIGN_IN_2FA,
-          TwoFactorAuthenticationSetting.DISABLED.getLabel());
+          .setValidationExpression("{oidcClients} empty or {sign_in_2fa} = 'Disabled'");
```

Comment:
Use `TwoFactorAuthenticationSetting.DISABLED.getLabel()` instead of the value directly